index.html

```diff
@@ -8218,6 +8200,10 @@
 										onclick="export_share_story()">Share</a>
 								</li>
 
+								<li class="nav-item" id="topbtn_settings">
+									<a class="nav-link" href="#" id="btn_aesthetics"
```

Comment:
Don't put this inside the main UI. If you need a customize button, place it within the aesthetic UI gear menu 
![image](https://github.com/LostRuins/lite.koboldai.net/assets/39025047/1f09227b-f097-4cac-8fa0-2936ce5f9acc)